### ISSUE TITLE
[SEO-MAGIC] Pages won't save // Few critical errors

### ISSUE THREAD

**Issue by cur1ou5soul** [getgrav/grav-premium-issues#87]
I Can't save a Page or delete all SEO magic information. 

System 
Dedicated Server Nginx

Grav 1.7.10.
SeoMagic 1.0.3

![image](https://user-images.githubusercontent.com/57013347/114901082-bcdbfb80-9e14-11eb-8868-265723f8d566.png)

If I disable the plugin the error doesn't pop up. 

`Too few arguments to function Grav\Plugin\SEOMagic\SEOGenerator::removeSEOData(), 0 passed in /var/www/html/siskaselement.com/user/plugins/seo-magic/seo-magic.php on line 151 and exactly 1 expected`

**Comment by rhukster** [comment]
Is it possible to get a copy of a page (the .md file) that is causing this issue, then I can see what's tripping things up?

**Comment by cur1ou5soul** [comment]
I dug a little deeper and found more or less a way to nail the error down

# I've set up my site to be multi Site
# I've Domains ending in .com and .de
# I redirected the /de to .de and /en .com 

That's the .en.md
```

---
title: 'Guiding Light [Around the World]'
feed:
    limit: 10
media_order: Cover_1500_sm.jpg
shop: d
spotify: d
published: true
visible: false
seo-magic:
    images:
        image_name: Cover_1500_sm.jpg
---

```

**Comment by cur1ou5soul** [comment]
It seems to me as if you view it in the admin panel, it can't be shown

![image](https://user-images.githubusercontent.com/57013347/114906615-82755d00-9e1a-11eb-8af0-ff5a35efd909.png)

That's the link. 
https://siskaselement.de/news/guiding-light-around-the-world

**Comment by cur1ou5soul** [comment]
I also get a 401 if I want to reinstall ;-)

**Comment by cur1ou5soul** [comment]
I disabled Flex and found this error:

![image](https://user-images.githubusercontent.com/57013347/114911761-9a031480-9e1f-11eb-9fa5-f8fc9cc6b03a.png)

**Comment by rhukster** [comment]
I think the problem is that the page URL is not visible or not accessible.  Granted this error needs to be handled better, but that appears to be the underlying issue.  Is this behind a firewall or perhaps authentication protected?  SEOMagic uses a web crawler to crawl the pages, so if the crawler cannot access the pages, it will be an issue.

**Comment by cur1ou5soul** [comment]
Well even when I'm inside my Home URL it won't work.

Is it possible to find the URL that the crawler uses?

**Comment by cur1ou5soul** [comment]
If I try to delete all SEO Magic files

![image](https://user-images.githubusercontent.com/57013347/114918600-8a87c980-9e27-11eb-8344-4c35a597b256.png)

**Comment by rhukster** [comment]
Let me see if i can address the errors and go from there.. i'll get back to you.

**Comment by cur1ou5soul** [comment]
I've got one more :-) 

![image](https://user-images.githubusercontent.com/57013347/114918879-e3576200-9e27-11eb-928f-ee4cbb95c6ad.png)

**Comment by rhukster** [comment]
I think that's a multi-site issue...

**Comment by rhukster** [comment]
I have released 1.0.4. Should be available in about 30 mins.   I'm not confident that it will address your underlying issue which I think is related to multisite, but should at least not throw errors.

**Comment by rhukster** [comment]
Closing based on latest release.. please reopen if you continue to have issues.
